src/main/java/me/sat7/dynamicshop/transactions/Sell.java

```diff
@@ -35,8 +35,16 @@ public static void quickSellItem(Player player, ItemStack myItem, String shopNam
             for (ItemStack item : player.getInventory().getContents()) {
                 if(item == null || item.getType() == null) continue;
                 if (item.getType() == myItem.getType()) {
-                    amount += item.getAmount();
-                    player.getInventory().removeItem(item);
+                	if (myItem.hasItemMeta()) {
+                		if(item.getItemMeta().equals(myItem.getItemMeta())) {
```

Comment:
instead of checking if the item meta equals the other meta, use the item.isSimilar() function